index.html

```diff
@@ -241,6 +241,57 @@ <h3>Skills</h3>
 
     </section>
 
+    <!----------------- form area -->\
+    <section id="form-area1">  
+       
+        <div class="txt">
+            <P class="form-txt">
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </P>
+        </div>
+
+              <form class="form-area" action="https://formspree.io/f/mjvdwrry" method="POST"> 
+                <form action="https://formspree.io/f/{form_id}" method="post">
+                <div class="user-details">
+                    
+                 
+                            <input type="text" placeholder="First name" required>
+                       
```

Comment:
- [ ] Earlier, the first reviewer requested that you apply HTML validations to the form elements. Kindly apply the `maxlength` attribute of `30` on the `input` element with `type="text"`. This is the required HTML validation for this element in this milestone. 👍 😎

Reply:
hi [Gambit142](https://github.com/Gambit142)
it resolved now

Reply:
hi [Gambit142](https://github.com/Gambit142)
kindly check and let me know thank for you reveiw